src/test/java/teammates/sqlui/webapi/GetInstructorPrivilegeActionTest.java

```diff
@@ -0,0 +1,231 @@
+        testInstructor2OfCourse1 = getTypicalInstructor();
+        testInstructor1OfCourse1.setCourse(c1);
+        testInstructor1OfCourse1.setGoogleId("user-googleId-1");
+        testInstructor2OfCourse1.setCourse(c1);
+        testInstructor2OfCourse1.setGoogleId("user-googleId-2");
+
+        InstructorPrivileges privileges = testInstructor1OfCourse1.getPrivileges();
+        // update section privilege for testing purpose.
+
+        // course level privilege
+        privileges.updatePrivilege(Const.InstructorPermissions.CAN_MODIFY_COURSE, true);
+        privileges.updatePrivilege(Const.InstructorPermissions.CAN_MODIFY_STUDENT, true);
+        privileges.updatePrivilege(Const.InstructorPermissions.CAN_MODIFY_SESSION, false);
+        privileges.updatePrivilege(Const.InstructorPermissions.CAN_MODIFY_INSTRUCTOR, false);
+
+        privileges.updatePrivilege(Const.InstructorPermissions.CAN_VIEW_STUDENT_IN_SECTIONS, false);
+
+        privileges.updatePrivilege(Const.InstructorPermissions.CAN_VIEW_SESSION_IN_SECTIONS, false);
+        privileges.updatePrivilege(Const.InstructorPermissions.CAN_SUBMIT_SESSION_IN_SECTIONS, false);
+        privileges.updatePrivilege(
+                Const.InstructorPermissions.CAN_MODIFY_SESSION_COMMENT_IN_SECTIONS, false);
+
+        // section level privilege
+        privileges.updatePrivilege(studentSectionName,
+                Const.InstructorPermissions.CAN_VIEW_STUDENT_IN_SECTIONS, true);
+        privileges.updatePrivilege(studentSectionName,
+                Const.InstructorPermissions.CAN_MODIFY_SESSION_COMMENT_IN_SECTIONS, true);
+
+        // session level privilege
+        privileges.updatePrivilege(studentSectionName, feedbackSessionName,
+                Const.InstructorPermissions.CAN_VIEW_SESSION_IN_SECTIONS, true);
+        privileges.updatePrivilege(studentSectionName, feedbackSessionName,
+                Const.InstructorPermissions.CAN_SUBMIT_SESSION_IN_SECTIONS, true);
+        privileges.updatePrivilege(studentSectionName, feedbackSessionName,
+                Const.InstructorPermissions.CAN_MODIFY_SESSION_COMMENT_IN_SECTIONS, true);
+
+        testInstructor1OfCourse1.setPrivileges(privileges);
+
+        testInstructor1OfCourse1.setRole(InstructorPermissionRole.INSTRUCTOR_PERMISSION_ROLE_CUSTOM);
+    }
+
+    @Test
+    void testAccessControl_unregisteredUsers_cannotAccess() {
+        logoutUser();
+        loginAsUnregistered(Const.ParamsNames.USER_ID);
+
+        String[] submissionParams = { Const.ParamsNames.COURSE_ID, "course_id" };
+        verifyCannotAccess(submissionParams);
+    }
+
+    @Test
+    void testAccessControl_students_cannotAccess() {
+        loginAsStudent(Const.ParamsNames.STUDENT_ID);
+
+        String[] submissionParams = { Const.ParamsNames.COURSE_ID, "course_id" };
+        verifyCannotAccess(submissionParams);
+    }
+
+    @Test
+    void testAccessControl_instructorsOfTheSameCourse_canAccess() {
```

Comment:
can we also add one more test to cover this block?

https://github.com/TEAMMATES/teammates/blob/4d91ef4738cd4e219abd7ba01505260098855170/src/main/java/teammates/ui/webapi/GetInstructorPrivilegeAction.java#L35-L39